### DISCUSSION TITLE
【建议】Champ-R的功能发展方向建议

### DISCUSSION THREAD

**Discussion by sugarnine9** [cangzhang/ChampR#159] [General]
不知道开发者有没有看过一款OPGG的插件，OP.GG Extension（http://www.op.gg/extension/），这是一款OPGG推出的浏览器插件，安装在浏览器上，设置好之后就可以在游戏开始后进行一些设置：
1. 多重搜索。在比赛开始前检查一下你的队友以往战绩、常用英雄、KDA等。
2. 英雄建议。它将为你选择的英雄提供明确的指导。
3. 自动符文设置。甚至不需要点击设置符文，我们会帮你完成。
4. 物品推荐。不需要再犹豫该买什么了。
5. 禁选助手。你想赢吗？那么就打开这个功能吧。

这5个功能里面有些是比较不错的，有些功能其他用户已经在issue里面提议过了，比如说英雄选择建议和禁选助手什么的。
开发者也可以开发自动设置符文的功能，优先选择OPGG的符文等，大乱斗的优先设置ARAM的符文。
召唤师技能不知道能不能自动设置，不过可以放在英雄头像旁边或者符文旁边。也可以加上技能加点推荐，比如潘森：Q>W>E。

以上就是一些我在使用OP.GG Extension浏览器插件的使用体验和心得，希望能够给开发者的更新提供一些思路。

再次感谢开发者！谢谢！

关于上述插件，不支持大陆国服，其他服可以使用。详情请看：【聊电Jing】LOL自动查队友战绩 & 自动配符文+出装推荐的超神插件! | Google Chrome扩充功能 OP.GG Extension  UP主：Jing_tw  https://www.bilibili.com/video/BV1N4411t7dP

**Comment by cangzhang** [comment]
感谢反馈, 我们会在之后优先加上_技能顺序_的提示

**Comment by sugarnine9** [comment]
> 感谢反馈, 我们会在之后优先加上_技能顺序_的提示

好滴，开发者加油！一直有在使用，感觉很舒服，脱离WeG的感觉真好！

**Comment by he429134733** [comment]
请问大佬能提供一个国内的下载地址么，在github上下载太搞心态了。而且介意把软件弄一个自动升级的功能

**Comment by cangzhang** [comment]
@he429134733 https://ngabbs.com/read.php?tid=21861241

**Comment by he429134733** [comment]
请问大佬，可以在哪里找到比较全的官方提供的api文档？

**Comment by zx1140165584** [comment]
大乱斗玩家，我挺想看见对面和己方总胜率和英雄胜率的，能更好的针对大哥。
